src/commands/ref_command_handler.js

```diff
@@ -34,22 +34,50 @@ function cppReference(message, args) {
                 }
             }
 
-
             const results = [];
             for (const ref of hrefs) {
                 if (ref.search(args[0]) > 0) {
                     console.log(ref);
                     results.push("https://en.cppreference.com/" + ref);
                 }
             }
+			
             if (results.length > 0) {
-                channel.send(results);
+                var errorMessage = "There are too many results, please use a more specific query";
+                var stringLength = 0;
+                var msg = `Results found for ${args[0]}:\n`;
+                for (var str of results) {
```

Comment:
Please do not use var
```suggestion
                for (const str of results) {
```